### PR TITLE
Package and mvn profile arrangements after repo ownership transfer

### PR DESCRIPTION
**Pull request by emrahkocaman**
This PR;
- Aligns package namings with other Hazelcast plugins.
- Adds necessary mvn profiles, checkstyle/findbugs/master/release)
- Fixes checkstyle/findbugs errors.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -231,6 +276,231 @@
...
+                <id>sign-artifacts</id>
+                <phase>verify</phase>
+                <goals>
+                  <goal>sign</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
```
Comment (noctarius) [inline pom.xml]: This overwrites the Javadoc configuration!

File: pom.xml [hunk 2]
```diff
@@ -231,6 +276,231 @@
...
+    <profile>
+      <id>release-snapshot</id>
+      <properties>
+        <javadoc>true</javadoc>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
```
Comment (noctarius) [inline pom.xml]: same here

File: README.adoc [hunk 3]
```diff
@@ -141,7 +140,7 @@ link:https://github.com/kubernetes/kubernetes/tree/v1.0.6/cluster/addons/dns[Kub
       <!-- activate the Kubernetes plugin -->
       <discovery-strategies>
         <discovery-strategy enabled="true"
-            class="com.noctarius.hazelcast.kubernetes.HazelcastKubernetesDiscoveryStrategy">
+            class="HazelcastKubernetesDiscoveryStrategy">
```
Comment (bilalyasar) [inline README.adoc]: it should be `com.hazelcast.HazelcastKubernetesDiscoveryStrategy`

File: pom.xml [hunk 4]
```diff
@@ -231,6 +276,231 @@
 
   <profiles>
     <profile>
+      <id>checkstyle</id>
```
Comment (mesutcelik) [inline pom.xml]: I am not sure we need separate profiles for checkstyle + findbugs.. They can be integrated into default build...

**Comment by mesutcelik** [comment]
👍

File: pom.xml (hunk 1 above)
Comment (emrahkocaman) [inline pom.xml]: @noctarius This is just like other plugins. I didn't get what's wrong here.

File: pom.xml (hunk 4 above)
Comment (emrahkocaman) [inline pom.xml]: I've removed separate profiles moved them to the default build.

File: README.adoc (hunk 3 above)
Comment (emrahkocaman) [inline README.adoc]: 👍

File: pom.xml (hunk 1 above)
Comment (emrahkocaman) [inline pom.xml]: @noctarius ?

**Comment by emrahkocaman** [comment]
Merging this, we can have a look to javadoc plugin later.

**Pull request merged**